lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -68,6 +68,12 @@
             - Toggle that controls if the machine is allocated/deallocated, only useful with state='present'.
         default: True
         type: bool
+    generalized:
```

Comment:
why 'd'?

Reply:
it follows other options, which are "started", "stopped".... etc....